tests/ui/const-generics/std/const-generics-range.rs

```diff
@@ -1,4 +1,5 @@
-// [full] check-pass
+// [full] known-bug: unknown
```

Comment:
Lol, I could add these derives to the range structs I guess? I didn't want to deal with it.

Reply:
Yeah it's probably best not to add `ConstParamTy` impls to everything in std, that probably should go through libs members lol